cfme/common/vm.py

```diff
@@ -472,8 +472,12 @@ def retire(self):
         self.load_details(refresh=True)
         lcl_btn(self.TO_RETIRE, invokes_alert=True)
         sel.handle_alert()
-        flash.assert_success_message(
-            "Retire initiated for 1 VM and Instance from the CFME Database")
```

Comment:
Personally I would prefer that we version pick the string, that way it's easier to add more if the versions change in future, and it's what pick was designed for.